### PR TITLE
Fixes for app user flow

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/atoms/Button/Button.js [hunk 1]
```diff
@@ -25,17 +25,18 @@ type ButtonProps = {|
   /** possible button types */
   type?: 'submit' | 'button',
   /** possible button colors */
-  color?: 'primary' | 'secondary' | 'neutral' | 'warning',
+  color?: 'primary' | 'secondary' | 'neutral' | 'warning' | 'brand' | 'marketing',
   /** the type of button */
   variant?: 'outlined' | 'raised',
   /** posible sizes */
-  size?: 'sm' | 'md' | 'lg',
+  size?: 'sm' | 'md' | 'lg' | 'xl',
```
Comment (ej9x) [inline src/atoms/Button/Button.js]: Then needs to add `xs` too

Comment (zouxuoz) [reply src/atoms/Button/Button.js]: Added

File: src/atoms/Button/Button.js [hunk 2]
```diff
@@ -25,17 +25,18 @@ type ButtonProps = {|
   /** possible button types */
   type?: 'submit' | 'button',
   /** possible button colors */
-  color?: 'primary' | 'secondary' | 'neutral' | 'warning',
+  color?: 'primary' | 'secondary' | 'neutral' | 'warning' | 'brand' | 'marketing',
```
[the remark this answers is not in the document]
Comment (zouxuoz) [reply src/atoms/Button/Button.js]: You are right, fixed

File: src/atoms/Grid/GridLayout.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { createStyledTag, createTheme } from '../../utils';
 type GridLayoutProps = {|
   children?: React$Node,
   /** possbile spaces between grid items */
-  gap?: 'xs' | 'md' | 'lg' | 'none',
+  gap?: 'xs' | 'md' | 'lg' | 'xl' | 'none',
```
[the remark this answers is not in the document]
Comment (zouxuoz) [reply src/atoms/Grid/GridLayout.js]: Added

File: src/theme/colors.js [hunk 4]
```diff
@@ -44,6 +44,7 @@ const COLORS = {
   SECONDARY: PALETTE.GREEN,
   SUCCESS: PALETTE.LIGHT_GREEN,
   DANGER: PALETTE.LIGHT_RED,
+  BRAND: '#E94235',
```
[the remark this answers is not in the document]
Comment (zouxuoz) [reply src/theme/colors.js]: Added

File: src/atoms/dataEntry/Input/Input.js [hunk 5]
```diff
@@ -23,7 +23,9 @@ type InputCommonProps = {
   maxLength?: number,
   /** callback to set input ref */
   insideRef?:(HTMLInputElement) => void,
-}
+  /** kind of the input */
+  kind?: 'app' | 'marketing',
```
[the remark this answers is not in the document]
Comment (ej9x) [reply src/atoms/dataEntry/Input/Input.js]: Then delete `marketing` from the Heading and Paragraph too.

File: src/atoms/typography/Link/Link.js [hunk 6]
```diff
@@ -5,26 +5,53 @@ import React from 'react';
...
+    },
+    color: {
+      primary: {
+        color: colors.PRIMARY_LINK_COLOR,
+      },
+      white: {
+        color: colors.WHITE,
+      },
+      brand: {
+        color: colors.BRAND,
```
Comment (ej9x) [inline src/atoms/typography/Link/Link.js]: After changes palette has no BRAND color

**Pull request merged**